AdminWebsite/AdminWebsite/ClientApp/src/tslint.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../tslint.json",
+  "linterOptions": {
+    "exclude": ["**/api-client.ts"]
```

Comment:
any reason we're doing this? We don't have this in place for the other apps

Reply:
something has changed recently with nswag and so the tslint disable and tslint disable are no longer the first lint and so lining keeps failing. I checked against a few versions and it keeps doing it. It's a file we don't really care to lint because it's autogenerated anyway